src/components/Places.js

```diff
@@ -0,0 +1,3 @@
+export default function Places() {
+  return <h2>Places</h2>;
```

Comment:
i think you can even return h1, since this word would be with the highest hierarchy